text/0000-optimize-by-default.md

```diff
@@ -0,0 +1,166 @@
+- Start Date: 2015-03-10
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Make rustc and cargo produce optimized binaries by default.
+
+# Motivation
+
+
+Optimizing by default is a safety & usability feature that everyone benefits
```

Comment:
I clarified the usage of the word "safety" in several places in the document. Thanks for the feedback!